### PR TITLE
Upgrade mx-agent jar for new metric naming to datadog

### PR DESCRIPTION
**Pull request by cwesdorp**
For DEP-715 upgrade the DD zip from CDN which has an updated version of the Mendix Agent to collect statistics. See https://github.com/mendix/mx-agent/releases/tag/mx-agent-v0.10.1

### CODE REVIEW

File: lib/datadog.py [hunk 1]
```diff
@@ -285,7 +286,7 @@ def compile(install_path, cache_dir):
     # aws s3 cp ../dd-vX.Y.Z.tar.gz s3://mx-cdn/mx-buildpack/experimental/
     buildpackutil.download_and_unpack(
         buildpackutil.get_blobstore_url(
-            "/mx-buildpack/experimental/dd-v0.9.0.tar.gz"
+            "/mx-buildpack/experimental/{}".format(DD_SIDECAR)
```
Comment (joshp123) [inline lib/datadog.py]: Do we have any recovery in place if this file does not exist on the CDN?

Comment (cwesdorp) [reply lib/datadog.py]: No, not yet. It's just a version bump now.

Comment (joshp123) [reply lib/datadog.py]: Seems a little risky; since I presume deploys will fail if we forget to upload this file. Is there a unit test that checks this?

**Comment by truimike** [comment]
Hoi, I get some errors with this new update of the mx-agent version. Especially that the .jar file is not present at the given file path. 

Log files are from the Azure container logs for the app services containers:
Command line to be used when starting the JVM: /build/.local/bin/java -Dfile.encoding=UTF-8 -Djava.io.tmpdir=/build/data/tmp -XX:OnError=kill -s USR1 5 -XX:OnOutOfMemoryError=kill -s USR2 5 -Xmx512M -Xms512M -XX:MaxMetaspaceSize=256M -Djava.locale.providers=JRE,SPI,CLDR -javaagent:/build/.local/datadog/**mx-agent-v0.10.1.jar** -Xbootclasspath/a:/build/.local/datadog/mx-agent-v0.10.1.jar -DMX_INSTALL_PATH=/build/runtimes/7.22.2.44474 -jar /build/runtimes/7.22.2.44474/runtime/launcher/runtimelauncher.jar /build/
2019-02-13T08:53:41.317359283Z Error opening zip file or JAR manifest missing : **/build/.local/datadog/mx-agent-v0.10.1.jar**
2019-02-13T08:53:41.317537588Z Error occurred during initialization of VM

**Comment by joshp123** [comment]
Hi @truimike, thanks for letting us know about this problem. I personally don't have enough knowledge here to see what's going on, so I will refer this to the relevant team member internally.

Our team's internal workflow usually manages incoming requests via our support team as opposed to GitHub issues, so to give you the best quality response here, can I ask you to also send an email to support@mendix.com, briefly detailing your problem? Feel free to cc me (Josh.Palmer@mendix.com) also. Thanks!

**Pull request merged**